mlir/test/lib/Interfaces/TilingInterface/TestTilingInterfaceTransformOps.td

```diff
@@ -71,11 +71,33 @@ def TestTileUsingForallOp : Op<Transform_Dialect, "test.tile_using_forall",
                       Variadic<TransformHandleTypeInterface>:$loops);
 
   let assemblyFormat = [{
-    $target ($tile_sizes^)? (`interchange` $interchange^)?
-    (`mapping` $mapping^)?
+    $target ($tile_sizes^)? (`interchange` `=` $interchange^)?
+    (`mapping` `=` $mapping^)?
     attr-dict `:` functional-type(operands, results)
   }];
 }
 
+def TestFuseUsingForallOp : Op<Transform_Dialect, "test.fuse_using_forall",
```

Comment:
maybe: `fuse_using_forall_greedily` ?

Reply:
just using this as a mirror for `fuse` op for testing. I want to fold this op into a proper transform op, I just dont know the final form of the operation. So maybe ill leave it as is for now?